### PR TITLE
Bottom sheet example review

### PR DESCRIPTION
**Pull request by Leehman**
Bottom sheet example review and docs import;

snap 2018-02-01 at 12 29 46

### CODE REVIEW

**Comment by Leehman** [comment]
Working to resolve items. My 'eslint' on desktop show none of these.

**Comment by Leehman** [comment]
Tested in desktop, tweaked case of names and UI.

File: examples/bottom-sheets/bottomGrid.vue [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="text-xs-center">
+    <v-bottom-sheet inset >
```
Comment (johnleider) [inline examples/bottom-sheets/bottomGrid.vue]: extra space after inset

File: examples/bottom-sheets/bottomGrid.vue [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      <v-btn
+        slot="activator"
+        color="purple"
+        dark>Click me Again</v-btn>
```
Comment (johnleider) [inline examples/bottom-sheets/bottomGrid.vue]: the `>` should go on its own line

File: examples/bottom-sheets/bottomGrid.vue [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      <v-layout
+        row
+        wrap>
```
Comment (johnleider) [inline examples/bottom-sheets/bottomGrid.vue]: the `>` should go on its own line

File: examples/bottom-sheets/bottomGrid.vue [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        <v-flex
+          xs3
+          v-for="tile in tiles"
+          :key="tile.title"
+          d-flex>
```
Comment (johnleider) [inline examples/bottom-sheets/bottomGrid.vue]: the `>` should go on its own line

File: examples/bottom-sheets/bottomGrid.vue [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+          <v-card
+            flat
+            tile>
+            <v-card-text>
+              <v-avatar
+                size="32px"
+                tile>
```
Comment (johnleider) [inline examples/bottom-sheets/bottomGrid.vue]: the `>` should go on its own line

File: examples/bottom-sheets/bottomGrid.vue [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+                <img
+                  :src="`/static/doc-images/bottom-sheets/${tile.img}`"
+                  height="90px"
+                  :alt="tile.title">
```
Comment (johnleider) [inline examples/bottom-sheets/bottomGrid.vue]: the `>` should go on its own line

File: examples/bottom-sheets/bottomGrid.vue [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+              </v-avatar>
+              <span class="hidden-sm-and-down">{{ tile.title }}</span>
+            </v-card-text>
+          </v-card>
+        </v-flex>
+      </v-layout>
+    </v-bottom-sheet>
+  </div>
+</template>
```
Comment (johnleider) [inline examples/bottom-sheets/bottomGrid.vue]: space between template and script

File: examples/bottom-sheets/bottomGrid.vue (hunk 1 above)
Comment (Leehman) [reply examples/bottom-sheets/bottomGrid.vue]: Fixed

File: examples/bottom-sheets/bottomGrid.vue (hunk 2 above)
Comment (Leehman) [reply examples/bottom-sheets/bottomGrid.vue]: Fixed

File: examples/bottom-sheets/bottomGrid.vue (hunk 3 above)
Comment (Leehman) [reply examples/bottom-sheets/bottomGrid.vue]: Fixed

File: examples/bottom-sheets/bottomGrid.vue (hunk 4 above)
Comment (Leehman) [reply examples/bottom-sheets/bottomGrid.vue]: Fixed

File: examples/bottom-sheets/bottomGrid.vue (hunk 5 above)
Comment (Leehman) [reply examples/bottom-sheets/bottomGrid.vue]: Fixed

File: examples/bottom-sheets/bottomGrid.vue (hunk 6 above)
Comment (Leehman) [reply examples/bottom-sheets/bottomGrid.vue]: Fixed

File: examples/bottom-sheets/bottomGrid.vue (hunk 7 above)
Comment (Leehman) [reply examples/bottom-sheets/bottomGrid.vue]: Fixed, now to push all the changes.

Comment (Leehman) [reply examples/bottom-sheets/bottomGrid.vue]: snap 2018-02-13 at 09 10 00

**Comment by Leehman** [comment]
snap 2018-02-14 at 09 02 01

**Comment by Leehman** [comment]
Ready to reviewed against v1.0:

snap 2018-02-16 at 09 31 30

File: examples/bottom-sheets/bottomGrid.vue [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+      >
+        <v-flex
+          xs3
+          v-for="tile in tiles"
+          :key="tile.title"
+          d-flex
+        >
+          <v-card
+            flat
+            tile>
```
Comment (johnleider) [inline examples/bottom-sheets/bottomGrid.vue]: `>` should be on next line

File: examples/bottom-sheets/bottomGrid.vue [hunk 9]
```diff
@@ -0,0 +1,61 @@
+<template>
+  <div class="text-xs-center">
+    <v-bottom-sheet inset>
+      <v-btn
+        slot="activator"
+        color="purple"
+        dark
+      >
+      Click Again</v-btn>
```
Comment (johnleider) [inline examples/bottom-sheets/bottomGrid.vue]: text should be indented and the closing `</v-btn>` on a new line

**Pull request closed without merging**